lib/DataIntegrityProof.js

```diff
@@ -351,8 +351,8 @@ export class DataIntegrityProof extends LinkedDataProof {
   async canonizeProof(proof, {documentLoader}) {
     // `proofValue` must not be included in the proof options
     proof = {
-      '@context': DATA_INTEGRITY_CONTEXT,
-      ...proof
+      ...proof,
+      '@context': DATA_INTEGRITY_CONTEXT
```

Comment:
This can be adjusted to align with the behaviour of jsonld-signatures with the following additional change:
```suggestion
      '@context': document['@context'] ?? DATA_INTEGRITY_CONTEXT
```
(`document` is available in the function's additional parameters)